From 71e30ce0cd823541264257f65d8ee08aa6ef6908 Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Mon, 20 Jul 2020 16:11:40 +0800
Subject: [PATCH 1/1] Test case for driver .h file

Description: Change the value of 'AvabilableHigh' in the struct defined in MdeModulePkg/Universal/CapsulePei/Capsule.h

Repo: edk2

Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 MdeModulePkg/Universal/CapsulePei/Capsule.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/MdeModulePkg/Universal/CapsulePei/Capsule.h b/MdeModulePkg/Universal/CapsulePei/Capsule.h
index 3d9cab02c485..d1f034a78f33 100644
--- a/MdeModulePkg/Universal/CapsulePei/Capsule.h
+++ b/MdeModulePkg/Universal/CapsulePei/Capsule.h
@@ -55,7 +55,7 @@ typedef union {
     UINT64  MustBeZero:2;             // Must Be Zero
     UINT64  Available:3;              // Available for use by system software
     UINT64  PageTableBaseAddress:40;  // Page Table Base Address
-    UINT64  AvabilableHigh:11;        // Available for use by system software
+    UINT64  AvabilableHigh:10;        // Available for use by system software
     UINT64  Nx:1;                     // No Execute bit
   } Bits;
   UINT64    Uint64;
-- 
2.27.0.windows.1

